.buildbot.sh

```diff
@@ -34,19 +36,26 @@ tmpdir=/tmp/lua-$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1
 mkdir -p $tmpdir
 cd $tmpdir
 	git clone --depth 1 -b v5.4.0 https://github.com/lua/lua.git .
-	make CC=$CC MYCFLAGS="-std=c99 -DLUA_USE_LINUX -fembed-bitcode -flto" MYLDFLAGS="-Wl,-E -flto -fembed-bitcode" AR="$AR rc" RANLIB=$RANLIB MYLIBS=" -ldl" -j 16
-	$LLVM_LINK *.o -o lua.linked.bc
+	# Dry run of `make` to list the commands executed
+	make -n CC=$CC MYCFLAGS="-std=c99 --config cheribsd-riscv64-purecap.cfg -c -emit-llvm" MYLDFLAGS="-Wl,-E -Wl,-v" AR="$AR rc" RANLIB=$RANLIB MYLIBS=" -ldl" -j 16 > make-commands.sh
+	# Remove conflicting `-march`
+	sed -i 's/-march=native//g' make-commands.sh
+	# Build `.bc` instead of `.o` files
+	sed -i 's/\.o/\.bc/g' make-commands.sh
+	sh make-commands.sh
+	echo $tmpdir
```

Comment:
I don't think the `echo` is doing anything useful?

Reply:
Oh my, I thought I had removed it... I'll fix it.

Reply:
Fix in abcbcac.